tests/test_algorithm.py

```diff
@@ -4479,6 +4481,10 @@ def handle_data(context, data):
                 context.order(context.sid(0), 10)
                 assert len(context.get_open_orders()) == 1
             elif today_session == first_asset_auto_close_date:
+                # We do not cancel open orders until the end of the auto close
+                # date, so our open order should still exist at this point.
+                assert len(context.get_open_orders()) == 1
+            elif today_session == first_asset_auto_close_date + trading_day:
```

Comment:
This might be more clear as `self.trading_calendar.next_session(first_asset_auto_close_date)`.